test/commands/mentor/request/lock_test.rb

```diff
@@ -61,7 +61,9 @@ class Mentor::Request::LockTest < ActiveSupport::TestCase
     # Sanity check: can lock when mentor has no existing locks
     Mentor::Request::Lock.(create(:mentor_request), mentor)
 
-    # Sanity check: can lock when mentor has one existing lock
+    # Sanity check: can lock when mentor has less than four existing locks
+    Mentor::Request::Lock.(create(:mentor_request), mentor)
+    Mentor::Request::Lock.(create(:mentor_request), mentor)
```

Comment:
```suggestion
    3.times { Mentor::Request::Lock.(create(:mentor_request), mentor) }
```

Maybe? But I'm fine with either :)